index.html

```diff
@@ -89,16 +89,16 @@
             class="rec-7"
           />
 
-          <h2 id="headline-text">I'm Kanza. Glad to see you!</h2>
-          <p id="supporting-text">
+          <h2 class="headline-text">I'm Kanza. Glad to see you!</h2>
```

Comment:
Kindly, I suggest that the main title is a unique H1 tag, the accessibility of your site will improve a lot.